backend/src/crd/controller/viewer/main.go

```diff
@@ -42,6 +43,9 @@ var (
 	maxNumViewers = flag.Int("max_num_viewers", 50,
 		"Maximum number of viewer instances allowed within "+
 			"the cluster before the controller starts deleting the oldest one.")
+	namespace = flag.String("namespace", "kubeflow",
```

Comment:
can we expose this through CRD container Dockerfile?
Here is an example
https://github.com/kubeflow/pipelines/blob/master/backend/Dockerfile.scheduledworkflow#L21
https://github.com/kubeflow/pipelines/blob/master/manifests/namespaced-install/ml-pipeline-scheduledworkflow-deployment-patch.yaml#L10

Reply:
Done. And the default namespace is set to be "kubeflow"